### ISSUE TITLE
The two large files in the example folder cannot be downloaded through LFS tool

### ISSUE THREAD

**Issue by moulai** [JGCRI/demeter#19]
Hello, I have cloned the Demeter and tried to run the example attached. However, the two large files in the example/inputs/observed and example/inputs/projected folders cannot be downloaded through the Git Large File Storage (LFS) tool. Only two csv files containing source file information can be downloaded.
Is there another way to download these two files?
Thanks so much!

**Comment by crvernon** [comment]
Hello @moulai !  We just released Demeter v1.2.0 which offers the ability to retrieve this data like the following:
```python
import demeter

# the directory that you want to download and extract the example data to
data_dir = "<my data download location>"

# download and unzip the package data to your local machine
demeter.get_package_data(data_dir)
```

You can also download the data from here:  https://zenodo.org/record/4773639#.YKWTPpNKhUM 

Please let us know if you have any further questions and thanks for using `demeter`!

**Comment by crvernon** [comment]
Addressed in PR #20

**Comment by moulai** [comment]
Thank you very much!
